src/wallet/rpcwallet.cpp

```diff
@@ -33,21 +33,34 @@
 
 #include <univalue.h>
 
-static const std::string WALLET_ENDPOINT_BASE = "/wallet/";
-
-CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest& request)
+void JSONRPCRequestWalletResolver(JSONRPCRequest& jreq, const HTTPRequest& httpreq)
```

Comment:
In commit "RPC: Pass wallet through JSONRPCRequest"

Multiwallet rpc calls are broken in this commit (and the whole PR) because JSONRPCRequestWalletResolver is not actually registered or called anywhere. Maybe a commit was lost in the rebase? The problem causes multiwallet.py test to fail with "Wallet file not specified"

Reply:
Fixed